app/assets/stylesheets/application.scss

```diff
@@ -69,14 +89,47 @@ body {
   transition: all 0.2s;
 }
 
+.big-checkbox {width: 20px; height: 20px;}
+
+.btn-submit {
+  font-size: 80%;
+  border-radius: 5rem;
+  letter-spacing: .1rem;
+  font-weight: bold;
+  padding: 1rem;
+  transition: all 0.2s;
+}
```

Comment:
we now have a general button class with the shape and styling as on the signin signup forms

Reply:
Nice Work!

Reply:
love it, I appreciate you going on the search for the buttons/roundness!